tests/main/quota-groups-systemd-accounting/task.yaml

```diff
@@ -0,0 +1,59 @@
+summary: Test to ensure that quota groups and systemd accounting is effective.
+
+details: |
+  Some older systemd's such as that of xenial, Amazon Linux 2, and Centos 7 
+  suffer from an unfortunate bug wherein the accounting for memory and tasks 
+  becomes broken and returns the maximum value for a 64-bit integer instead of
+  the real value. This test checks that either the system works and does not
+  trigger the accounting bug, or that we properly don't attempt to use quota 
+  groups on systems that are affected.
```

Comment:
the "don't attempt" part seems to be tested in tests/main/snap-quota not here?

Reply:
Maybe that part of the description coould be dropped from this test and added to the other one?

Reply:
yeah sorry I wrote the descritption before I decided to move the bit that checks that systems that we don't support quota groups on are tested only once in tests/main/snap-quota instead of multiple times here, in tests/main/snap-quota-groups and in tests/main/snap-quota

Reply:
I adjusted the description of the test to make clear that we test those buggy systems in tests/main/snap-quota instead of here